tutorials/angular/start/stocks/src/app/stock-details/stock-details.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from "@angular/core";
+import { DataService } from '../data.service';
+import { Stock, GlueStatus } from '../types';
+import { GlueService } from '../glue.service';
+
+@Component({
+    templateUrl: './stock-details.component.html'
+})
+export class StockDetailsComponent implements OnInit {
+
+    public stock: Stock;
+    public glueStatus: GlueStatus = "disconnected";
+    public clientMessage: string;
+

```

Comment:
```suggestion
  private glueSubscription: Glue42Web.Interop.Subscription;
```
Because it is used in the tutorial guide.

Reply:
added